charts/xui-webapp/values.yaml

```diff
@@ -55,8 +55,8 @@ nodejs:
     SERVICES_CCD_CASE_ASSIGNMENT_API: http://aac-manage-case-assignment-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     SERVICES_LOCATION_API: http://rd-location-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     # Services Hearings endpoints
-    SERVICE_HEARINGS_COMPONENT_API:
-      sscs: http://sscs-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal,
+    SERVICES_HEARINGS_COMPONENT_API:
+      sscs: http://sscs-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
I see how this is working now I've looked at the log file Ernest provided. It will have to be 
SERVICES_HEARINGS_COMPONENT_API_SSCS: http:// whatever